app/lib/forum_support/categories.rb

```diff
@@ -10,7 +10,7 @@ def self.included(base)
     end
 
     def index
-      @categories = @forum.categories :include => :topics
+      @categories = @forum.categories.includes(:topics)
```

Comment:
I've looking for an explanation to why this doesn't work anymore. It is related to n+1 queries, isn't it?
The tests pass with `@forum.categories` alone, so I can't be sure this is actually the correct fix.... 🤷🏻‍♂️

Reply:
When did it stop working? I thnk that `includes` is the standard way to do and its purpose is to avoid N+1 queries. But even without it, things should work, just more queries will be performed.

Did data type of `categories` change or there was something upgraded?

Reply:
Apparently `include =>` is outdated now and some tests will fail after upgrading to rails 5. See any of the test logs that are in the description.